aggregate_enclave_scripts/setup_agg.bash

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+SCONE_MODE=AUTO
+pip3 install paste
```

Comment:
do we need paste? what for?

Reply:
If I remember correctly, it was a multithreaded python server.

Reply:
It is used in the run() command at the bottom of the aggregator.py and controller.py files.

Reply:
but you are now using bottle, so we should be able to remove it?

Reply:
But we are not using enclaves anymore and can put paste in the docker image if need be.

Reply:
ah I see. The main e-mission server uses `cherrypy` as the multithreaded server in bottle's `run`. I would suggest using the same WSGI-compliant server in both. Do you remember why you picked paste instead of cherrypy?

Reply:
The urap 2018 aggregate queries chat on Signal has our convo about it but I don't specifically remember. "the issue with bottle was that it was too fast, not too slow. they needed to change the protocol to handle responses from the enclaves to be more robust to quick execution" but if we are not using enclaves anymore, then bottle being "too fast" will be more of an issue. Let me try the same bottle run() as the e-mission webserver.